backend/typescript/models/entity.pgmodel.ts

```diff
@@ -18,4 +18,7 @@ export default class Entity extends Model {
 
   @Column
   bool_field!: boolean;
+
+  @Column
+  file_name!: string;
```

Comment:
The reason why this has ! and mongo model does not have required = true is because the field in mongo can't be set to "" if it's required, whereas in Postgres it can